client/blocks/all-sites/style.scss

```
@@ -6,6 +6,13 @@
 	padding: 0;
 	position: relative;
 
+	.all-sites__title {
+		display: flex;
+		.all-sites__title-chevron-icon {
```

[NITPICK, NON-BLOCKER] Can we add an extra line space here?

```suggestion
		display: flex;
		
		.all-sites__title-chevron-icon {
```